### PR TITLE
Do not panic when template fails to execute

### PR DESCRIPTION
**Pull request by sthaha**
PipelineRun describe command panics when templates fails to execute.
This patch changes the behaviour to write errors to stderr when
template execution fails

Signed-Off-By: sthaha@redhat.com

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [🙅‍♂] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added) Existing tests cover the change
- [🙅‍♂ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
release-note
```

### CODE REVIEW

**Comment by sthaha** [comment]
@vdemeester do you know why `CLA` seems broken?

**Comment by hrishin** [comment]
@sthaha old CLA is not working for me. I didn't sign new CLA.

**Comment by hrishin** [comment]
I think it has introduced Linux Foundation CLA as well. Is it @vdemeester ?

File: pkg/cmd/pipelinerun/describe.go [hunk 1]
```diff
@@ -130,13 +133,14 @@ func printPipelineRunDescription(out io.Writer, prname string, p cli.Params) err
...
 	}
 
-	w := tabwriter.NewWriter(out, 0, 5, 3, ' ', tabwriter.TabIndent)
+	w := tabwriter.NewWriter(s.Out, 0, 5, 3, ' ', tabwriter.TabIndent)
 	t := template.Must(template.New("Describe Pipelinerun").Funcs(funcMap).Parse(templ))
+
 	err = t.Execute(w, data)
 	if err != nil {
-		panic(err)
+		fmt.Fprintf(s.Err, "Failed to execute template")
```
Comment (hrishin) [inline pkg/cmd/pipelinerun/describe.go]: Failed -> failed? as we are following lowercase everywhere?

**Comment by vdemeester** [comment]
@sthaha @hrishin yeah, CLA switch to Linux Foundation CLA on Friday

**Comment by sthaha** [comment]
I signed it

**Comment by vdemeester** [comment]
@sthaha needs a rebase :angel:

**Pull request merged**